inesita.gemspec

```diff
@@ -15,14 +15,14 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.require_paths = ['lib']
 
-  s.add_dependency 'opal', '>= 0.9', '< 0.11'
+  s.add_dependency 'opal-sprockets', '~> 0.4.0.0.11.0.rc1.3.1.beta'
```

Comment:
@fazibear the idea of the crazy version number of opal-sprockets is that one should state the desired version of opal and optionally of sprockets and let rubygems/bundler pick the right version of opal sprockets, so ideally should look like this:

```rb
s.add_dependency 'opal', '>= 0.10.4', '< 0.12'
s.add_dependency 'opal-sprockets'
```